gon/linear.py

```diff
@@ -1,36 +1,25 @@
-from enum import (IntEnum,
-                  unique)
-from functools import partial
-from typing import Union
+from typing import (Tuple,
+                    Union)
 
 from reprit.base import generate_repr
+from robust.linear import (SegmentsRelationship,
+                           segment_contains,
+                           segments_relationship)
 
+from gon.hints import Coordinate
 from .angular import (Angle,
-                      AngleKind,
                       Orientation)
 from .base import Point
 
+SegmentsRelationship = SegmentsRelationship
```

Comment:
it will "anchor" this object to the current module so PyCharm for example will suggest it to import from, because we don't want everywhere in project to import from `robust` package since it violates DRY and may cause a lot of problems in case of changes